Assets/Scripts/NursePatrol.cs

```diff
@@ -60,9 +63,22 @@ void GotoNextPoint()
     // Update is called once per frame
     void Update()
     {
+
+        float distance = Vector3.Distance(transform.position, spy.transform.position);
```

Comment:
Wait so does this mean that even if we're behind the enemy, they can start chasing us?
I thought like only once we're within their sights (so like the pink sight line we can see) they will chase us? Not sure what the design we agreed on is though so I may be wrong.

Reply:
OOOF that's true... i'm not sure what our design decision was exactly but it does make more sense if we can sneak past them behind their back